support/troubleshooting/investigating-monitoring-issues.adoc

```diff
@@ -9,7 +9,7 @@ toc::[]
 {product-title} includes a preconfigured, preinstalled, and self-updating monitoring stack that provides monitoring for core platform components. In {product-title} {product-version}, cluster administrators can optionally enable monitoring for user-defined projects.
 
 // Note - please update the following sentence if you add further modules to this assembly.
-You can follow these procedures if your own metrics are unavailable or if Prometheus is consuming a lot of disk space.
+You can follow these procedures if your own metrics are unavailable, if Prometheus is consuming a lot of disk space, or if the `KubePersistentVolumeFillingUp` alert is firing for Prometheus.
```

Comment:
```suggestion
Use the following these procedures:
* If your own metrics are unavailable.
* If Prometheus is consuming a lot of disk space. 
* If the `KubePersistentVolumeFillingUp` alert is firing for Prometheus.
```
For scannability

Reply:
Hello @ShaunaDiaz i applied all the changes, I just tweaked the last change so that it does not include incomplete sentences:

Use these procedures if the following issues occur:

* Your own metrics are unavailable.
* Prometheus is consuming a lot of disk space. 
* The `KubePersistentVolumeFillingUp` alert is firing for Prometheus.

I hope that is okay with you. Thank you!